### PR TITLE
[15.0][IMP] maintenance_account: Create equipment description from move line name

### PR DESCRIPTION
**Pull request by carolinafernandez-tecnativa**
- The first line of the name from account.move should go to the name of the equipment.
- The rest of the lines go to the equipment description

@Tecnativa
TT47802
@pedrobaeza @victoralmau

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This is not what you have to do.

- The first line of the `name` from `account.move` should go to the name of the equipment.
- The rest of the lines go to the description.

**Comment by carolinafernandez-tecnativa** [comment]
> This is not what you have to do.
> 
> * The first line of the `name` from `account.move` should go to the name of the equipment.
> * The rest of the lines go to the description.

Changes applied.

File: maintenance_account/models/account_move.py [hunk 1]
```diff
@@ -115,11 +116,12 @@ def _set_equipment_category(self):
...
             "move_line_id": self.id,
-            "name": self.product_id.name,
+            "name": self.name.split('\n')[0],
             "product_id": self.product_id.id,
             "category_id": self.equipment_category_id.id,
             "assign_date": self.move_id.date,
             "effective_date": self.move_id.date,
             "partner_id": self.move_id.partner_id.id,
             "partner_ref": self.move_id.ref,
+            "note": self.name.split('\n', 1)[-1],
```
Comment (pedrobaeza) [inline maintenance_account/models/account_move.py]: We have to take the remaining lines, not just one.

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**